### PR TITLE
CSIC-43 Support for RAW volume

### PR DESCRIPTION
**Pull request by nikhilbarge**
PR For
[Create PV workflows for iSCSI protocol](https://jira.infinidat.com/browse/CSIC-40)
[Delete PV workflows for iSCSI protocol](https://jira.infinidat.com/browse/CSIC-41)
[Chap auth](https://jira.infinidat.com/browse/CSIC-42)
[Support for RAW volume (iSCSI)](https://jira.infinidat.com/browse/CSIC-43)

Added examples (yaml)

### CODE REVIEW

**Review by dhanashreebiradar — APPROVED** [review]
looks good

File: deploy/helm/infinibox-csi-driver/templates/controllerserver.yaml [hunk 1]
```diff
@@ -146,8 +151,10 @@ spec:
             - name: socket-dir
               mountPath: /var/run/csi      
         - name: driver                 
-          image: {{ required "Provide the Infinibox csi driver container image." .Values.images.csidriver }}
-          imagePullPolicy: Always
+          # image: {{ required "Provide the Infinibox csi driver container image." .Values.images.csidriver }}
+          # imagePullPolicy: Always
+          image: nikhilbarge/infinidat-csi-driver:test1
```
Comment (gadekarnitesh) [inline deploy/helm/infinibox-csi-driver/templates/controllerserver.yaml]: remove "nikhilbarge/infinidat-csi-driver:test1" path

File: deploy/helm/infinibox-csi-driver/templates/nodeserver.yaml [hunk 2]
```diff
@@ -70,16 +70,18 @@ spec:
             capabilities:
               add: ["SYS_MODULE"]
             allowPrivilegeEscalation: true          
-          image: {{ required "Provide the Infinibox csi driver container image." .Values.images.csidriver }}
-          imagePullPolicy: Always
+          #image: {{ required "Provide the Infinibox csi driver container image." .Values.images.csidriver }}
+          #imagePullPolicy: Always
```
Comment (gadekarnitesh) [inline deploy/helm/infinibox-csi-driver/templates/nodeserver.yaml]: remove path "nikhilbarge/infinidat-csi-driver:test1"

File: deploy/helm/infinibox-csi-driver/templates/secret-infinibox.yaml [hunk 3]
```diff
@@ -18,3 +18,41 @@ data:
   password: {{ required "Password is required!" .Values.Infinibox.password }}
   {{- end }}
   hosturl: "{{ .Values.Infinibox.hosturl | b64enc }}"
+
+# {{- range  $key, $infisecrets := .Values.InfiniboxSecrets }}
+# apiVersion: v1
+# # some: problem section
+# # {{ $infisecrets | quote }}
+# kind: Secret
+# metadata:
```
Comment (gadekarnitesh) [inline deploy/helm/infinibox-csi-driver/templates/secret-infinibox.yaml]: remove commented code

File: deploy/examples/iscsi/rawpvc.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# apiVersion: v1
+# kind: PersistentVolumeClaim
+# metadata:
+#   name: ibox-pvc-raw-demo
+# spec:
+#   volumeMode: Block
```
Comment (gadekarnitesh) [inline deploy/examples/iscsi/rawpvc.yaml]: remove commented code

**Review by gadekarnitesh — COMMENTED** [review]
remove unused code

**Review by gadekarnitesh — APPROVED** [review]
looks good

**Pull request merged**